src/pages/instances/InstanceList.tsx

```diff
@@ -405,6 +423,21 @@ const InstanceList: FC = () => {
                         )
                       }
                     />
+                    <InstancePagination
+                      currentPage={currentPage}
+                      setCurrentPage={(val) => setCurrentPage(val)}
```

Comment:
_[I remember...](https://github.com/canonical/lxd-ui/pull/121#discussion_r1054194980)_
 
> Due to the beauty of lambdas, this can be written shorter as

`setCurrentPage={setCurrentPage}`
The same applies to the `setPageSize` prop below.

:wink:

Reply:
We can, but the type we receive (and need to define) in `InstancePagination` gets ugly. Because  `setCurrentPage`  is coming as second argument from a `useState`. Hence, this tiny lambda to make the forwarded type simpler.

Reply:
It's not coming from `useState`, it's coming from `usePagination`. And it has the correct type already. I checked the change locally before proposing it and there is no need to change anything else.

Reply:
Huh, that is a surprise to me. Good to know!